internal/tests/api/with-ns-registered/minipool_count_test.go

```diff
@@ -0,0 +1,39 @@
+package with_ns_registered
+
+import (
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	hdtesting "github.com/nodeset-org/hyperdrive-daemon/testing"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	nodeWalletAddressString string = "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"
+	goodPassword            string = "some_password123"
+)
+
+var (
+	nodeWalletAddress common.Address = common.HexToAddress(nodeWalletAddressString)
```

Comment:
You don't actually need this, there's a global var in `main_test` called `nodeAddress` that provides it for you

Reply:
updated